### DISCUSSION TITLE
Pinned windows

### DISCUSSION THREAD

**Discussion by telk5093** [OpenRCT2/OpenRCT2#15674] [Feature requests]
It will be great to be implemented the **pinned windows** to prevent closing all windows by DELETE key. (like OpenTTD). And assign an another hotkey to close all windows including pinned windows. If it is duplicated feature request, then sorry.

<img width="365" alt="schermafbeelding 2016-04-22 om 15 55 55" src="https://cloud.githubusercontent.com/assets/7860305/14743660/f5eec87e-08a2-11e6-9da2-3c5ddff196f8.png">

**Comment by IntelOrca** [comment]
This has always been on the list of things to implement. But I think its better to wait for the window system to be re-designed so we don't have to go through every single window and adding a new widget.

**Comment by telk5093** [comment]
Okay. I'll be wait it :)

**Comment by DoveCannon** [comment]
I would also like this feature.

**Comment by pantsmodetoo** [comment]
+1'ing this, ten years later.
